src/Persistence.LinnApps/ServiceDbContext.cs

```diff
@@ -801,5 +808,37 @@ private static void BuildPotentialMoveDetails(ModelBuilder builder)
             e.Property(s => s.PalletNumber).HasColumnName("PALLET_NUMBER");
             e.Property(s => s.SernosNumber).HasColumnName("SERNOS_NUMBER");
         }
+
+        private static void BuildWorkstations(ModelBuilder builder)
+        {
+            var e = builder.Entity<Workstation>().ToTable("WORK_STATION");
+            e.HasKey(l => new { l.WorkstationCode });
```

Comment:
don't need the anonymous type here, i.e simple single field keys can just be e.HasKey(l => l.WorkStationCode)